client/aws/errors.go

```diff
@@ -3,12 +3,14 @@ package aws
 import "github.com/juju/errgo"
 
 const (
-	AlreadyAssociated       = "Resource.AlreadyAssociated"
-	BucketAlreadyExists     = "BucketAlreadyExists"
-	BucketAlreadyOwnedByYou = "BucketAlreadyOwnedByYou"
-	InvalidSubnetConflict   = "InvalidSubnet.Conflict"
-	KeyPairDuplicate        = "InvalidKeyPair.Duplicate"
-	SecurityGroupDuplicate  = "InvalidGroup.Duplicate"
+	AlreadyAssociated        = "Resource.AlreadyAssociated"
+	InvalidSubnetConflict    = "InvalidSubnet.Conflict"
+	BucketAlreadyExists      = "BucketAlreadyExists"
+	BucketAlreadyOwnedByYou  = "BucketAlreadyOwnedByYou"
+	KeyPairDuplicate         = "InvalidKeyPair.Duplicate"
+	SecurityGroupDuplicate   = "InvalidGroup.Duplicate"
+	ELBAlreadyExists         = "DuplicateLoadBalancerName"
+	ELBConfigurationMismatch = "already exists and it is configured with different parameters"
```

Comment:
This is the lack of uniformity of what AWS API returns, we don't have any control about it. Sad but true.